modules/add-pod-namespace-to-sensor-and-admission-control.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-from-45.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="add-pod-namespace-to-sensor-and-admission-control_{context}"]
+= Add `POD_NAMESPACE` to `sensor` and `admission-control` deployments
+
+[role="_abstract"]
+You must update the sensor and admission-control deployment .yaml files to set the `POD_NAMESPACE` environment variable.
```

Comment:
Do we want to say this is required when upgrading from a version earlier than 4.6? (not super clear on this - it's only for upgrades from a version earlier than 4.6 to 4.6?)

example:

When upgrading from a version earlier than 4.6, you must update the `sensor` and `admission-control` deployment .yaml files to set the `POD_NAMESPACE` environment variable.